src/routes/landing/components/CitySelector.js

```diff
@@ -76,10 +122,45 @@ class CitySelector extends React.PureComponent<PropsType> {
     }, [])
   }
 
+  renderListByLocation (): React.Node {
+    if (this.props.currentLatitude === null || this.props.currentLongitude === null) {
+      return null
+    }
+    const currentLatitude = this.props.currentLatitude
+    const currentLongitude = this.props.currentLongitude
+    let cities = this.props.cities.filter(_city => _city.live)
+      .sort((a: CityModel, b: CityModel) => compareDistance(a, b, currentLongitude, currentLatitude))
+    const numberOfClosestCities = 3
```

Comment:
Maybe move constants to the start of the file (e.g. `NUMBER_OF_CLOSEST_CITIES = 3`)